src/providers/MeetingProvider/docs/MeetingProvider.stories.mdx

```diff
@@ -188,6 +188,10 @@ const Root = () => {
 ```
 For more information on `VideoPriorityBasedPolicy`, check Amazon Chime JS SDK [priority downlink policy guide](https://aws.github.io/amazon-chime-sdk-js/modules/prioritybased_downlink_policy.html).
 
+#### reconnectTimeoutMs
+
+The `reconnectTimeoutMs` specifies how long to wait after the meeting connection has been lost before trying to reconnect to the meeting.
```

Comment:
Request to check : https://github.com/aws/amazon-chime-sdk-component-library-react/pull/697/files#r769061196

Up to you, but would be good to link the FAQ on reconnection here as a hyperlink:
https://aws.github.io/amazon-chime-sdk-js/modules/faqs.html#what-is-the-timeout-for-connect-and-reconnect-and-where-can-i-configure-the-value